### DISCUSSION TITLE
Org mode / Org Roam support?

### DISCUSSION THREAD

**Discussion by parabolapersuader** [gsantner/markor#1745] [Ideas] [answered]
Org Roam is a superset of Org mode, so for most purposes Org mode should be enough. And, although there are differences with Markdown, the differences are smaller than between say ZIM and Markdown.

**Comment by gsantner** [comment, marked as answer]
https://github.com/gsantner/markor/issues/182

Go ahead and do it, if you can.

**Reply by parabolapersuader** [reply]
Oh, thank you for the reminder of the sh** atmosphere of the Open Source.

**Reply by gsantner** [reply]
I do spend all my time in my leisure time here, just like contributors. And everybody decides when, how much and what they want to do, or not do.    
If you want to help, do that. Open source projects also need people giving, not only people receiving & leeching. And - there is no place for rioting here.

**Reply by harshad1** [reply]
@parabolapersuader You could also pay someone to add this functionality if you so desired.

**Reply by cmhobbs** [reply]
Where's the best place to start looking in the source code to maybe try to add org-mode support?  I know a little Java but haven't built an android application before.  It might be a fun thing to tinker with.

**Reply by gsantner** [reply]
By luck there is currently one new format in first addition phase at #1876 . So to get a grasp on what files to touch/add and what they roughly are about, I think the merge request gives you perfect insights.

For already added formats check i.e. the Plaintext or Markor folders.

**Comment by ghost** [comment]
I am actually considering adding `org-mode` support, even though it is likely not going to be fast, as I know nothing about parsing. But in case something changes, I would like to subscribe to this thread.
